src/Extensions/AzureBlobStorage/AzureBlob/AzureBlobEgressProvider.cs

```diff
@@ -80,74 +77,6 @@ public AzureBlobEgressProvider(ILogger logger)
             }
         }
 
-        public async Task<string> EgressAsync(
```

Comment:
I think I would prefer that we keep this variant of the `EgressAsync` method because it supports writing blocks rather than requiring the entire stream to be written. This helps in failure modes where we can write out partial artifacts (e.g. logs, metrics, etc). @wiktork, do you have an opinion on which one is kept?

Reply:
I switched this out to use the other method - let me know if anything doesn't look right or if you decide you want to go back to the prior way.